### PR TITLE
Add Dutch(nl)

### PR DESCRIPTION
**Pull request by jvanaert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ n2words(123, {lang: 'es'})   // 'ciento veintitrés'
 - `ko` (Korean)
 - `lt` (Lithuanian)
 - `lv` (Latvian)
+- `nl` (Nederlands)
```
Comment (forzagreen) [inline README.md:65]:
```suggestion
- `nl` (Dutch)
```
ISO language name, from https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

File: package.json [hunk 2]
```diff
@@ -38,7 +38,8 @@
     "latvian",
     "arabic",
     "hebrew",
-    "korean"
+    "korean",
+    "nederlands"
```
Comment (forzagreen) [inline package.json:42]:
```suggestion
    "dutch"
```

File: src/n2words.mjs [hunk 3]
```diff
@@ -26,7 +27,7 @@ import Num2Word_UK from './i18n/UK.mjs';
  */
 export default function(n, options) {
   var lang = 'EN';        // default language
-  var supportedLanguages = ['en', 'fr', 'es', 'de', 'pt', 'it', 'tr', 'ru', 'cz', 'no', 'dk', 'pl', 'uk', 'lt', 'lv', 'ar', 'he', 'ko'];
+  var supportedLanguages = ['en', 'fr', 'es', 'de', 'pt', 'it', 'tr', 'ru', 'cz', 'no', 'dk', 'pl', 'uk', 'lt', 'lv', 'ar', 'he', 'ko','nl'];
```
Comment (forzagreen) [inline src/n2words.mjs:30]: Please run linter: `npm run lint`

File: README.md (hunk 1 above)
Comment (jvanaert) [reply README.md]: ok, fixed

**Comment by jvanaert** [comment]
I tried the lint command, but it didn't do anything.  I guess you meant there was a space missing ?

**Comment by MeekLogic** [comment]
> I tried the lint command, but it didn't do anything.  I guess you meant there was a space missing ?

It should have given you output in your terminal. 

To have ESLint automatically fix things you should be able to run something like

`eslint --fix lib/**/*.mjs`

**Comment by jvanaert** [comment]
when I run `npx eslint --fix src/**/*.mjs --debug` it prints`eslint:source-code-fixer No fixes to apply` for every file.
What is the linter output you are expecting ?

**Comment by forzagreen** [comment]
I'll take care of the linting.
Thank you very much !

**Pull request merged**